site/en/docs/extensions/mv3/sandboxingEval/index.md

```diff
@@ -64,42 +64,94 @@ named "sandbox.html". The manifest entry looks like this:
 ### Load the sandboxed file {: #load_file }
 
 In order to do something interesting with the sandboxed file, we need to load it in a context where
-it can be addressed by the extension's code. Here, [sandbox.html][8] has been loaded into the
-extension's [Event Page][9] ([eventpage.html][10]) via an `iframe`. [eventpage.js][11] contains code
-that sends a message into the sandbox whenever the browser action is clicked by finding the `iframe`
+it can be addressed by the extension's code. Here, sandbox.html has been loaded into
+an extension page via an `iframe`. The page's javascript file contains code that sends a message 
```

Comment:
Wouldn't it be better to use an offscreen document for this use case? Opening a new tab just for rendering a template feels like a really bad UX.

Reply:
@sebastianbenz I remember Devlin had an issue with this, but don't remember the specific concern. Should we wait for his answer on this one?

Reply:
Makes sense, can you please add me to the thread

Reply:
Thinking more about this, I think it's fine to keep as is. The templating sample makes actually more sense on a HTML page and the basic idea is independent of where it is implemented.